mewarpx/mewarpx/assemblies.py

```diff
@@ -25,15 +24,14 @@ class Assembly(object):
     # This is overridden if a diagnostic is installed to record scraped
     # current.
     scraper_diag = None
+    # List of diagnostic functions to call after collecting scraped particles
+    scraper_diag_fnlist = None
```

Comment:
This could be initialized as an empty list in __init__ as an alternative; saves you a clause later on.